src/app/content/components/Page/PageComponent.tsx

```diff
@@ -195,10 +195,10 @@ export default class PageComponent extends Component<PagePropTypes> {
       a.addEventListener('click', handler);
     });
 
-    mapSolutions(this.container.current, (button) => {
-      const handler = toggleSolution(button, this.props.intl);
-      this.clickListeners.set(button, handler);
-      button.addEventListener('click', handler);
+    mapSolutions(this.container.current, (summary) => {
+      const handler = toggleSolution(summary, this.props.intl);
+      this.clickListeners.set(summary, handler);
+      summary.addEventListener('toggle', handler);
```

Comment:
the `listenersOff` behavior is looking for `click` only and won't properly remove this

Reply:
actually we shouldn't need any js behavior to handle this at all, details/summary naturally handles expand collapse